src/diffusers/pipeline_utils.py

```diff
@@ -57,6 +58,10 @@ class DiffusionPipeline(ConfigMixin):
 
     config_name = "model_index.json"
 
+    def __init__(self):
```

Comment:
Could we remove the `__init__` function here? It's usually much easier to read components when there is no `__init__` function :-)

Reply:
The main reason here is that adding an `__init__` to a class makes the class automatically harder to understand e.g. if you abstract from this class and the class doesn't have an init you're given much more freedom with your "parent" class